packages/gasket-nextjs/src/server/request.js

```diff
@@ -0,0 +1,29 @@
+import { cookies, headers } from 'next/headers';
+
+const reqCache = new Map();
+
+// TODO: move this to `@gasket/nextjs` package
```

Comment:
I think you satisfied the TODO. Nit: doc block

Reply:
lolz - I had this in an internal package to start with. Good catch